cmd/glasskube/cmd/configure.go

```diff
@@ -74,28 +83,34 @@ func runConfigure(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(os.Stderr, "⚠️  Some values can not be resolved: %v\n", err)
 	}
 
-	if !cliutils.YesNoPrompt("Continue?", true) {
-		cancel()
+	if !configureCmdOptions.DryRun {
+		if !cliutils.YesNoPrompt("Continue?", true) {
+			cancel()
+		}
 	}
 
 	switch pkg := pkg.(type) {
 	case *v1alpha1.ClusterPackage:
+		values := pkg.Spec.Values
```

Comment:
There is another bug here: because maps are always pointers in go, `values` will also get overwritten by re-fetching the package. You can change it to `maps.Clone(pkg.Spec.Values)` then it should work.

Reply:
I missed this, while I try to update the configuration it seemed to work fine though, thanks @kosmoz